spring-ai-spring-boot-autoconfigure/pom.xml

```diff
@@ -18,6 +18,13 @@
 		<connection>git://github.com/spring-projects/spring-ai.git</connection>
 		<developerConnection>git@github.com:spring-projects/spring-ai.git</developerConnection>
 	</scm>
+	<repositories>
+		<repository>
+			<id>gemfire-release-repo</id>
```

Comment:
I need help getting this to work, ink particular in a CI environment.

Reply:
We can add credentials for the private repo to settings.xml, will that work for the CI environment?